src/base/bittorrent/tracker.cpp

```diff
@@ -29,15 +29,23 @@
  * Contact : chris@qbittorrent.org
  */
 
+#include <limits>
 #include <vector>
 #include <libtorrent/bencode.hpp>
 #include <libtorrent/entry.hpp>
 
+#include <boost/asio/ip/address.hpp>
+#if defined(BOOST_ASIO_DYN_LINK)
+#include <boost/asio/impl/src.hpp>
```

Comment:
@glassez 
The problem turned out to be that some boost asio dynamic libraries could be compiled without rvalue reference move support. All Travis-ci comilation attempts would fail without this inclusion. So I disabled the generation of moves. Thanks for pointing it out.